packages/idealo.design/src/server/db.js

```diff
@@ -54,12 +54,7 @@ export async function storeSinglePost({
 }
 
 export async function updateSinglePost(blog) {
-  const updatedPost = await sql`
-    update blogposts set ${
-      sql(blog, 'text', 'title' )
-  } where 
-    id = ${ blog.id }
-`     
+  const updatedPost = await sql`update blogposts set ${ sql(blog, 'text', 'title', 'categoryDisplayValue', 'categorySlug')} where  id = ${ blog.id }`
```

Comment:
Sorry, i'm not fine with this change. Maybe it was your editor trying to reformat the code, but for better readability it should be spread around multiple lines instead of one long line of code.

Reply:
I reformatted it back.